galley/pkg/config/analysis/analyzers/testdata/serviceroleservices.yaml

```diff
@@ -0,0 +1,111 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: details
+  namespace: default
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: reviews
+  namespace: default
+---
+apiVersion: rbac.istio.io/v1alpha1
+kind: ServiceRole
+metadata:
+  name: service-role-services-bogus-fqdn
+  namespace: default
+spec:
+  rules:
+    - services: ["bogus.default.svc.cluster.local"]
+      methods: ["GET", "HEAD"]
+---
+apiVersion: rbac.istio.io/v1alpha1
+kind: ServiceRole
+metadata:
+  name: service-role-services
```

Comment:
Please annotate this file with comments about what should / shouldn't succeed, see other testdata files for examples.